Assets/GameAssets/Boss/BossScripts/Attacks/BaseBossAttack.cs

```diff
@@ -48,6 +53,40 @@ protected virtual void BeginAttack()
         _isAttackActive = true;
     }
 
+    protected virtual void PlayIdleLoop(float second)
+    {
+        if (_idleSFXCoroutine == null)
+        _idleSFXCoroutine = StartCoroutine(LoopIdleCorrectfloat(second));
+    }
+    protected virtual void DestroyIdleLoop()
+    {
+        if (_idleSFXCoroutine != null)
+        {
+            StopCoroutine(_idleSFXCoroutine);
+        }
+    }
+    /// <summary>
+    /// Play heartbeat at controlled rate 
+    /// </summary>
+    /// <returns></returns>
+    private IEnumerator LoopIdleCorrectfloat (float second)
```

Comment:
This name is confusing to me. Also the F in float should be uppercase, but if you rename it then nvm. Perhaps we could get a more descriptive name than seconds. Maybe loopTime?